graylog2-server/src/main/java/org/graylog2/rest/resources/system/responses/KafkaJournalConfigurationSummary.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * This file is part of Graylog.
+ *
+ * Graylog is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Graylog is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.graylog2.rest.resources.system.responses;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.auto.value.AutoValue;
+import org.graylog.autovalue.WithBeanGetter;
+import org.graylog2.plugin.KafkaJournalConfiguration;
+import org.joda.time.Duration;
+
+import java.nio.file.Path;
+
+@AutoValue
+@WithBeanGetter
+@JsonAutoDetect
+public abstract class KafkaJournalConfigurationSummary {
+    private static final String FIELD_DIRECTORY = "directory";
+    private static final String FIELD_SEGMENT_SIZE = "segment_size";
+    private static final String FIELD_SEGMENT_AGE = "segment_age";
+    private static final String FIELD_MAX_SIZE = "max_size";
+    private static final String FIELD_MAX_AGE = "max_age";
+    private static final String FIELD_FLUSH_INTERVAL = "flush_interval";
+    private static final String FIELD_FLUSH_AGE = "flush_age";
```

Comment:
I wonder if we should include the formerly exposed fields `bin_dir`, `data_dir` and `plugin_dir` so we don't change the API?

Reply:
I think it's okay to change the API for this. The path data was never supposed to be part of the response and also doesn't make much sense in that context.

Reply:
Fine with me, I added the api change label, just in case